### DISCUSSION TITLE
【ZZZ】有没有玩亚服的朋友🧑‍🤝‍🧑

### DISCUSSION THREAD

**Discussion by harrysun23w** [share121/inter-knot#5133] [绝区零]
来加好友！每天上线好友栏空空的感觉好孤单😭
UID 1304963366
![IMG_4817](https://github.com/user-attachments/assets/cc302b4f-e7cd-42ce-b46c-303077037062)

**Comment by win2045** [comment]
这还不简单，自己创一个小号再加自己好友不就不空了嘛？（疑惑脸）

**Reply by harrysun23w** [reply]
天才！

**Comment by 1cupFor-deadintheafternoon** [comment]
亚服来++++++1300012506

**Reply by harrysun23w** [reply]
➕啦

**Comment by Andrea0126** [comment]
來了！1306382307
![img-1725796459345db277c8dfab32597244128f10c33abc20b83a3e03651e9ecb7ccc47a4ee7db3d](https://github.com/user-attachments/assets/b19ff218-b39f-4217-aee7-030cf6cd37fd)

**Reply by harrysun23w** [reply]
好耶！刚看到加了

**Reply by 1cupFor-deadintheafternoon** [reply]
我也加了，那个十四时刻是我

**Comment by Anslate** [comment]
来啦1300069551

**Reply by harrysun23w** [reply]
加了！

**Comment by obai-gs** [comment]
你们的亚服怎么多我一个数字
